core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -153,14 +151,15 @@ object TestUtils extends Logging {
     enableDeleteTopic: Boolean = false,
     interBrokerSecurityProtocol: Option[SecurityProtocol] = None,
     trustStoreFile: Option[File] = None,
+    saslProperties: Option[Properties] = None,
```

Comment:
Agree that there are too many parameters already. But a similar pattern is used for producers and consumers as well. Rather than change them all to call additional methods, leaving as-is for now. This keeps it consistent with the way SSL properties are set.